perfkitbenchmarker/providers/rackspace/flags.py

```diff
@@ -14,37 +14,39 @@
 
 from perfkitbenchmarker import flags
 
-flags.DEFINE_boolean('boot_from_cbs_volume', 'False',
-                     'When flag is included the instance will use a remote disk'
-                     'as its boot disk, if machine_type supports it.')
+flags.DEFINE_string('rack_path',
+                    default='rack',
+                    help='The path for the rack CLI binary.')
 
-flags.DEFINE_string('nova_path',
-                    'nova',
-                    'The path for the rackspace-novaclient tool.')
+flags.DEFINE_string('region', default='IAD',
+                    help='A string indicating which Rackspace region to use.')
 
-flags.DEFINE_string('neutron_path',
-                    'neutron',
-                    'The path for the rackspace-neutronclient tool.')
+flags.DEFINE_string('profile', default=None,
```

Comment:
Same applies to this and the other Rackspace flags. I think they should be prefixed with `rackspace_` unless they're actually provider-neutral. (I know that's not true of all provider-specific flags, but as far as I know we're not adding any new unprefixed ones.)